_posts/2023-05-17-save-date-ugm2023.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Save the Date - MDAnalysis UGM 2023
+---
+
```

Comment:
Do we have a nice image?

Reply:
Added an image of Lisbon with an MDA banner logo. Can't seem to get rid of the large spacing between the image and start of the text though